packages/paste-core/components/in-page-navigation/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import {render} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {InPageNavigation, InPageNavigationItem} from '../src';
+
+describe('InPageNavigation', () => {
+  it('should render a nav with correct aria-label', () => {
+    const {getByRole} = render(
+      <InPageNavigation aria-label="my-nav">
+        <InPageNavigationItem href="#">page 1</InPageNavigationItem>
+        <InPageNavigationItem href="#">page 2</InPageNavigationItem>
+      </InPageNavigation>
+    );
+
+    expect(getByRole('navigation').getAttribute('aria-label')).toBe('my-nav');
+  });
+
+  it('should render a list with list items', () => {
```

Comment:
Suggestion(non-blocking): could this render test and the "should render anchors" test be combined? that would reduce renders

Reply:
yes yes, good idea